From 5adca573c65177efb04d1ff4df3d6c666c37823f Mon Sep 17 00:00:00 2001
From: Maximiliano Sandoval R <msandova@protonmail.com>
Date: Tue, 17 Nov 2020 18:28:55 +0100
Subject: [PATCH 1/2] Metainfo: Revert license to CC0-1.0

GPL-3.0 or GPL-3.0-or-later are not valid licenses for metainfo files,
the latter not being considered code.

Reverts: 295049fe93d051e246c7c8dc53e6324664482da1

Fixes: https://github.com/lutris/lutris/issues/3277
Fixes (Downstream): https://github.com/flathub/net.lutris.Lutris/pull/127
---
 share/metainfo/net.lutris.Lutris.metainfo.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/share/metainfo/net.lutris.Lutris.metainfo.xml b/share/metainfo/net.lutris.Lutris.metainfo.xml
index 981db3915..8b969c27b 100644
--- a/share/metainfo/net.lutris.Lutris.metainfo.xml
+++ b/share/metainfo/net.lutris.Lutris.metainfo.xml
@@ -3,7 +3,7 @@
 <component type="desktop">
   <id>net.lutris.Lutris</id>
   <project_license>GPL-3.0-or-later</project_license>
-  <metadata_license>GPL-3.0-or-later</metadata_license>
+  <metadata_license>CC0-1.0</metadata_license>
   <translation type="gettext">lutris</translation>
   <developer_name translatable="no">Lutris Team</developer_name>
   <update_contact>strider@lutris.net</update_contact>

From 92e7e932365a87ca89568523c3669386a4546c65 Mon Sep 17 00:00:00 2001
From: Maximiliano Sandoval R <msandova@protonmail.com>
Date: Tue, 17 Nov 2020 18:35:19 +0100
Subject: [PATCH 2/2] debian/copyright: Add license details to metainfo

- Adds location of license.

Reverts: 813c64cc40681a9b6bb2046ab1dd825da2c2ce25
---
 debian/copyright | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/copyright b/debian/copyright
index f63f5ccf6..3502d4a72 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -8,3 +8,9 @@ Copyright: 2009-2020 Mathieu Comandon <strider@strycore.com>
 License: GPL-3.0-or-later
  On Debian systems, the complete text of the General Public License version 3
  can be found in "/usr/share/common-licenses/GPL-3".
+
+Files: share/metainfo/net.lutris.Lutris.metainfo.xml
+Copyright: Lutris Team <strider@lutris.net>
+License: CC0-1.0
+ On Debian systems, the complete text of the CC0-1.0 license
+ can be found in "/usr/share/common-licenses/CC0-1.0".
